ETC.tsv

```diff
@@ -14772,62 +14772,62 @@ ETC_20151001_014773	{img emoticon_0010 30 30} Surprised
 ETC_20151001_014774	{img emoticon_0016 30 30} Crying
 ETC_20151001_014775	{img emoticon_0007 30 30} Love
 ETC_20151001_014776	{img emoticon_0006 30 30} Hello
-ETC_20151001_014777	
-ETC_20151001_014778	
-ETC_20151001_014779	
-ETC_20151001_014780	
-ETC_20151001_014781	
-ETC_20151001_014782	
-ETC_20151001_014783	
-ETC_20151001_014784	
-ETC_20151001_014785	
-ETC_20151001_014786	
-ETC_20151001_014787	
-ETC_20151001_014788	
-ETC_20151001_014789	
-ETC_20151001_014790	
-ETC_20151001_014791	
-ETC_20151001_014792	
-ETC_20151001_014793	
-ETC_20151001_014794	
-ETC_20151001_014795	
-ETC_20151001_014796	
-ETC_20151001_014797	
-ETC_20151001_014798	
-ETC_20151001_014799	
-ETC_20151001_014800	
-ETC_20151001_014801	
-ETC_20151001_014802	
-ETC_20151001_014803	
-ETC_20151001_014804	
-ETC_20151001_014805	
-ETC_20151001_014806	
-ETC_20151001_014807	
-ETC_20151001_014808	
-ETC_20151001_014809	
-ETC_20151001_014810	
-ETC_20151001_014811	
-ETC_20151001_014812	
-ETC_20151001_014813	
-ETC_20151001_014814	
-ETC_20151001_014815	
-ETC_20151001_014816	
-ETC_20151001_014817	
-ETC_20151001_014818	
-ETC_20151001_014819	
-ETC_20151001_014820	
-ETC_20151001_014821	
-ETC_20151001_014822	
-ETC_20151001_014823	
-ETC_20151001_014824	
-ETC_20151001_014825	
-ETC_20151001_014826	
-ETC_20151001_014827	
-ETC_20151001_014828	
-ETC_20151001_014829	
-ETC_20151001_014830	
-ETC_20151001_014831	
-ETC_20151001_014832	
+ETC_20151001_014777	A conditional sentence selected should be true at all times.
+ETC_20151001_014778	A conditional sentence selected should be true at all times. Please refer to follow information.
+ETC_20151001_014779	A conditional sentence selected should be true at all times. Close the program.
+ETC_20151001_014780	Failed to table initialization. Please refer to follow information,  file load failure and past logs.
+ETC_20151001_014781	Failed to server initialization. Close the program.
+ETC_20151001_014782	Failed to Nexon AuthCheck. Please check the error code table refer to the error code and authCheck.h.
```

Comment:
Nexon AuthCheck has failed. Please check the error code table that corresponds with the error code of authCheck.h.